solvers/deep_mdd.py

```diff
@@ -25,7 +30,11 @@ class Solver(DeepDASolver):
     # the cross product for each key in the dictionary.
     # All parameters 'p' defined here are available as 'self.p'.
     default_param_grid = {
-        'criterion__reg': [1e-5, 1e-4, 1e-3, 1e-2, 1e-1, 1, 1e1, 1e2, 1e3],
+        'criterion__reg': [1e-3, 1e-2, 1e-1],
+        'criterion__adapt_criterion': [
+            MDDLoss(gamma=gamma)
+            for gamma in [2., 3.]
+        ],
```

Comment:
Why not gamma in [2, 4] ? In the paper, the best param for Office31 is gamma = 4.
But ok, it should not change everything ^^

Reply:
it's a matter of 0.05 points of accuracy, so forget about this